Frontend/ui-customer/src/Components/DriverInfoCard/View.purs

```diff
@@ -586,7 +696,15 @@ waitTimeView push state =
       , text state.data.waitingTime
       , lineHeight "24"
       , gravity CENTER
-      , color Color.black800
+      , color (case split (Pattern ":") state.data.waitingTime of
+             [minutes, _] ->
+               if minutes == "00 " 
+               then Color.green900  
+               else if minutes == "01 " || minutes == "02 "
+                    then Color.yellow900 
+                    else Color.orange800   
+             _ -> Color.black 
+          )
```

Comment:
don't do like this, create a function in `common.js` which will have arguments date and format, there you can send `mm` to get minutes

Reply:
Dont required ,we r already getting waiting time state data variable